### ISSUE TITLE
Upgrade abcjs_midi.js?

### ISSUE THREAD

**Issue by chrisspen** [paulrosen/abcjs#677]
Is abcjs_midi.js being updated anymore? 

I see the note in the README about "The file abcjs version supporting midi.js is the last version of the old style of sound production that will receive updates.", however, the link is broken, and I can't find any other documentation for how to upgrade old coding use abcjs_midi.js to the equivalent current code.

**Comment by paulrosen** [comment]
Sorry - I moved the old versions out of the repo since they are available with npm versioning. Also I created a separate repo to store them. So the link you are looking for is: https://github.com/paulrosen/historical-abcjs-versions/blob/main/version-6/abcjs_midi-min.js

Thanks for letting me know about the broken link! I'll fix that.

That's a good point about having a migration guide between the midijs version and the new synth. I've put that on my list and will get to it as soon as I can. If you want to post the code you are currently using I'll give you some pointers to upgrade.
